libctru/include/3ds/applets/miiselector.h

```diff
@@ -81,12 +93,94 @@ typedef struct
  */
 Result miiSelectorLaunch(const MiiSelectorConf *conf, MiiSelectorReturn* returnbuf);
 
+/**
+ * @brief Sets title of the Mii selector library applet
+ *
+ * @param conf Pointer to miiSelector configuration
+ * @param text Title text of Mii selector
+ */
+void miiSelectorSetTitle(MiiSelectorConf *conf, const char* text);
+
+/**
+ * @brief Specifies which special options are enabled in the Mii selector
+ *
+ * @param conf Pointer to miiSelector configuration
+ * @param options Options bitmask
+ */
+void miiSelectorSetOptions(MiiSelectorConf *conf, u32 options);
+
+/**
+ * @brief Specifies which guest Miis will be selectable
+ *
+ * @param conf Pointer to miiSelector configuration
+ * @param index Index of the guest Miis that will be whitelisted.
+ * @ref MIISELECTOR_GUESTMII_SLOTS can be used to whitelist all the guest Miis.
+ */
+void miiSelectorWhitelistGuestMii(MiiSelectorConf *conf, u32 index);
+
+/**
+ * @brief Specifies which guest Miis will be unselectable
+ *
+ * @param conf Pointer to miiSelector configuration
+ * @param index Index of the guest Miis that will be blacklisted.
+ * @ref MIISELECTOR_GUESTMII_SLOTS can be used to blacklist all the guest Miis.
+ */
+void miiSelectorBlacklistGuestMii(MiiSelectorConf *conf, u32 index);
+
+/**
+ * @brief Specifies which user Miis will be selectable
+ *
+ * @param conf Pointer to miiSelector configuration
+ * @param index Index of the user Miis that will be whitelisted.
+ * @ref MIISELECTOR_USERMII_SLOTS can be used to whitlist all the user Miis
+ */
+void miiSelectorWhitelistUserMii(MiiSelectorConf *conf, u32 index);
+
+/**
+ * @brief Specifies which user Miis will be selectable
+ *
+ * @param conf Pointer to miiSelector configuration
+ * @param index Index of the user Miis that will be blacklisted.
+ * @ref MIISELECTOR_USERMII_SLOTS can be used to blacklist all the user Miis
+ */
+void miiSelectorBlacklistUserMii(MiiSelectorConf *conf, u32 index);
+
+/**
+ * @brief Specifies which Mii the cursor should start from
+ *
+ * @param conf Pointer to miiSelector configuration
+ * @param index Indexed number of the Mii that the cursor will start on.
+ * If there is no mii with that index, the the cursor will start at the Mii
+ * with the index 0 (the personal Mii).
+ */
+void miiSelectorSetInitalIndex(MiiSelectorConf *conf, u32 index);
```

Comment:
This should be `miiSelectorSetInit**i**alIndex` (and also static inline, see comment in C file)